code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -191,13 +192,18 @@ var/veil_thickness = CULT_PROLOGUE
 				O.target_sacrificed = TRUE
 				veil_thickness = CULT_ACT_III
 				emergency_shuttle.force_shutdown()//No shuttle calls until the cult either wins or fails.
-				ticker.StartThematic("endgame")
 				spawn_bloodstones(A)
+				spawn(50)
```

Comment:
If you're doing spawns and sleeps for seconds, do 5 SECONDS for readability (Seconds just multiplies the value by 10)